docs/framework/winforms/disable-dpi-awareness-visual-studio.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Disable DPI-awareness in Visual Studio
+ms.date: 08/14/2018
+ms.prod: visual-studio-dev15
+ms.technology: vs-ide-designers
+ms.topic: conceptual
+author: gewarren
+ms.author: gewarren
+manager: douge
+---
+# Disable DPI-awareness in Visual Studio
+
+Visual Studio is a dots per inch (DPI) aware application, which means displays scale automatically. If an application states that it's not DPI-aware, the operating system scales the application as a bitmap. This behavior is also called DPI virtualization. The application still thinks that it's running at 100% scaling, or 96 dpi.
+
+## Rendering issue
+
+The **Windows Forms Designer** in Visual Studio doesn't currently have scaling support. This can cause display issues when you open Windows Forms apps in Visual Studio on HDPI monitors.
+
+For example, the first screenshot that follows shows **Windows Forms Designer** on a monitor at 100% scaling. The second screenshot shows what the form looks like when it's opened on a high dots per inch (HDPI) monitor:
+
+![Windows Forms Designer scaling at 100%](media/scaling-100-percent.png)
+
+![Windows Forms Designer on HDPI monitor](media/win-forms-designer-hdpi.png)
+
+In Visual Studio 2017 version 15.8 and later, when you open a Windows Form in the designer on an HDPI monitor, Visual Studio displays a yellow message bar at the top of the designer:
+
+![Scaling on your monitor yellow message bar in Visual Studio](media/scaling-gold-bar.png)
+
+The message reads **Scaling on your main display is set to 200%. This might cause rendering problems in the designer window.**
+
+## To resolve the problem
+
+There are two options to resolve the display problem:
+
+- Set your display scaling setting to 100%.
+
+   To do this in Windows 10, type **display settings** in the Cortana search box, and then select **Change display settings**. In the **Settings** window, set **Change the size of text, apps, and other items** to **100%**.
+
+- Select the option on the yellow message bar to restart Visual Studio as a DPI-unaware process.
+
+   > [!NOTE]
+   > - If you had undocked tool windows in Visual Studio when you selected the option to restart as a DPI-unaware process, the position of those tool windows may change.
+   > - If you use the default Visual Basic profile, or if you have the **Save new projects when created** option deselected in **Tools** > **Options** > **Projects and Solutions**, Visual Studio cannot reopen your project when it restarts as a DPI-unaware process. However, you can open the project by selecting it under **File** > **Recent Projects and Solutions**.
+
+   If you close and reopen Visual Studio when it's running in DPI-unaware mode, it becomes DPI-aware again. It's important to restart Visual Studio as a DPI-aware process when you're finished working in the **Windows Forms Designer**.
+
```

Comment:
Windows team had shared a published registry key that can be used to mark process as dpi-unaware:
https://blogs.technet.microsoft.com/askperf/2007/10/05/the-program-compatibility-assistant-part-two/
 
Could you please add this key as an alternative resolution?

KEY: 
    HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers
VALUE (REG_SZ): 
    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\devenv.exe"="DPIUNAWARE"
Where the path should be edited to match the actual path. This is the same reg key as is used by the "compatibility property page in file explorer.

Reply:
@Tanya-Solyanik  I thought we are not allowed to publish on docs the info about changing reg key since it is for internal use only. @merriemcgaw  Could you comment on it?

Reply:
This is a different key, here is the email from the WSG team:

From: Jeff Bisset 
Sent: Monday, July 2, 2018 6:04 PM
To: Merrie McGaw <Merrie.McGaw@microsoft.com>; Nicolas Leroux <nileroux@microsoft.com>; Rohit Bansal <rohitba@microsoft.com>; Abi Bennet <abbennet@microsoft.com>; klaus@loeffelmann.de; James Clarke (WDG) <James.Clarke@microsoft.com>; Leonardo Blanco <Leonardo.Blanco@microsoft.com>
Cc: Tanya Solyanik <Tatiana.Solyanik@microsoft.com>; Olia Gavrysh <Olia.Gavrysh@microsoft.com>
Subject: Re: DPI awareness Registry Key
 
Sorry about the slow response on this.  
 
Try setting this alternate registry value instead.  This should force devenv to run unaware.  Adjust path to the exe as appropriate.
 
KEY: 
    HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers
VALUE (REG_SZ): 
    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\devenv.exe"="DPIUNAWARE"
 
This the same registry setting that the app compat UI would have modified, if it were available. *The structure of the registry values behind the app compat UI are not documented per-se, but they are widely known and I would not have any qualms about telling people about them.  e.g. Do a bing search for "HIGHDPIAWARE layers" and you will find plenty of discussion.*
 
I don't know how/why exactly, but the app compat UI doesn't show for Office and some other Microsoft apps.  VS must be caught up in that.  I think that this boils down to reducing support costs by not letting people run Office etc. in odd modes.

Reply:
@Tanya-Solyanik is correct - this one we were given the thumbs up to publish.

Reply:
Perhaps we should explain why it is important to restart as a DPI aware project?  It might be obvious to the users once they're in there but if they're reading the whole article it might be nice to know that the rest of VS might be blurry.